.prettierignore

```diff
@@ -0,0 +1,3 @@
+.vscode/
+public/demo/data/
+src/assets/demo/flags/flags.css
```

Comment:
Just confirmed via [prettier docs on ignoring](https://prettier.io/docs/ignore) that it picks up `.gitignore` when present, so we only need to add files here that are in the repository (since they likely won't be in .gitignore).